tests/unittests/CoreGraphics.drawing/CGContextBlendModeTests.cpp

```diff
@@ -131,7 +131,8 @@ static void __drawRightStackPink(CGContextRef context, CGRect bounds) {
     CGContextRestoreGState(context);
 }
 
-class CGContextBlendMode : public WhiteBackgroundTest<>, public ::testing::WithParamInterface<::testing::tuple<bool, CGNamedBlendMode>> {
+class CGContextBlendMode : public WhiteBackgroundTest<PixelByPixelImageComparator<PixelComparisonModeDifferenceLimen<100>>>,
```

Comment:
It was something like 4-5 off.  I don't think we need to get that granular with our difference images.